Formula/x264.rb

```diff
@@ -35,14 +36,6 @@ class X264 < Formula
     fails_with :clang
   end
 
-  # update config.* and configure: add Apple Silicon support.
-  # upstream PR https://code.videolan.org/videolan/x264/-/merge_requests/35
-  # Can be removed once it gets merged into stable branch
-  patch do
-    url "https://code.videolan.org/videolan/x264/-/commit/eb95c2965299ba5b8598e2388d71b02e23c9fba7.diff?full_index=1"
-    sha256 "7cdc60cffa8f3004837ba0c63c8422fbadaf96ccedb41e505607ead2691d49b9"
-  end
-
   def install
     # Work around Xcode 11 clang bug
     # https://bitbucket.org/multicoreware/x265/issues/514/wrong-code-generated-on-macos-1015
```

Comment:
I believe we don't need this workaround as well. It was fixed in Xcode

Reply:
OK, it seems to build and test OK with XCode 12.1 on my box. Thanks much!